e2e/ChannelApeClient.spec.ts

```diff
@@ -684,4 +716,32 @@ describe('ChannelApe Client', () => {
     expect(channel.updatedAt.getUTCMilliseconds())
       .to.be.greaterThan(expectedCreatedAt.getUTCMilliseconds());
   }
+
+  /**
+   * Generates a random number based on the provided min and max values.
+   * @param {number} min Minimum number to generate (inclusive)
+   * @param {number} max Maximum number to generate (inclusive)
+   */
+  function randomNumber(min: number, max: number): number {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+
+  /**
+   * Generates a random semi-readable word based on the provided min an max length values.
+   * @param {number} min Minium word length (inclusive)
+   * @param {number} max Maximum word length (inclusive)
+   */
+  function randomWord(min: number, max: number): string {
```

Comment:
We already have fake word generator. Please see the order creation test in this class.

Please remove this `randomWord` function and replace calls to this function with calls to the `faker` sdk we already have in place.

Reply:
@rjdavis3 Didn't know about faker, so much better than what I had!